### ISSUE TITLE
Some links in README.md are not found

### ISSUE THREAD

**Issue by shenxianpeng** [docker/buildx#2843]
The following links [docker buildx install](https://github.com/docker/buildx/blob/master/docs/reference/buildx_install.md) and [docker buildx uninstall](https://github.com/docker/buildx/blob/master/docs/reference/buildx_uninstall.md) in README.md are not found. 

> Running the command [docker buildx install](https://github.com/docker/buildx/blob/master/docs/reference/buildx_install.md) sets up docker builder command as an alias to docker buildx build. This results in the ability to have docker build use the current buildx builder.

> To remove this alias, run [docker buildx uninstall](https://github.com/docker/buildx/blob/master/docs/reference/buildx_uninstall.md).

**Comment by crazy-max** [comment]
I think this relates to a change in docs generation where hidden commands don't generate documentation anymore, which makes sense imo:
* https://github.com/docker/buildx/blob/695200c81a4db45eb3d04c770391144dcc2a3c2c/commands/install.go#L50
* https://github.com/docker/buildx/blob/695200c81a4db45eb3d04c770391144dcc2a3c2c/commands/uninstall.go#L56

@dvdksn I wonder if we could just remove this section from the README: https://github.com/docker/buildx?tab=readme-ov-file#set-buildx-as-the-default-builder and add it to https://docs.docker.com/build/concepts/overview/#buildx instead without links to cli ref?

**Comment by dvdksn** [comment]
@crazy-max yeah we can do that. I could also clarify the discrepancy between a vanilla `docker build` and a `docker buildx build` w.r.t. default builder.

**Comment by crazy-max** [comment]
> I could also clarify the discrepancy between a vanilla `docker build` and a `docker buildx build` w.r.t. default builder.

Ah true we should explain this. I see this content that is almost correct: https://docs.docker.com/build/concepts/overview/#buildx

> Buildx and BuildKit are both installed with Docker Desktop and Docker Engine out-of-the-box. When you invoke the `docker build` command, you're using Buildx to run a build using the default BuildKit bundled with Docker.

We could add here how builder selection works when `docker build` is invoked vs `docker buildx build`.

**Comment by AbhishekJupalli-1409** [comment]
I am also interested in this issue . Allow me to collaborate with you
